src/Microsoft.SqlTools.ServiceLayer/ObjectExplorer/Nodes/NodeTypes.cs

```diff
@@ -1,147 +1,143 @@
-﻿//
+
+//
 // Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 //
 
+// This file was generated by a T4 Template. Do not modify directly, instead update the SmoTreeNodesDefinition.xml file
```

Comment:
I assume we didn't lose any manual edits when regenerating?

Reply:
I did do a check and have manually added those types back